src/js/components/invoice/InvoiceListTable.jsx

```diff
@@ -145,7 +146,7 @@ const InvoiceListTable = ({
             <button
               type="button"
               className="dropdown-item"
-              onClick={() => downloadInvoices()}
+              onClick={downloadInvoices}
```

Comment:
I said it once to @alannadolny , but we need to be aware, that in this case, if we added any argument to the `downloadInvoices` method, this would be run with the `event` from the button.